.editorconfig

```diff
@@ -20,7 +20,7 @@ continuation_indent_size = 4
 wildcard_import_limit = 999
 ij_kotlin_name_count_to_use_star_import = 999
 ij_kotlin_name_count_to_use_star_import_for_members = 999
-ktlint_standard_property-naming = disabled
+#ktlint_standard_property-naming = disabled
```

Comment:
For the disabled ones, let's just delete them in the future :)

Reply:
I'll follow this in the next linting issue and fix this one then. Thanks for your review.😊